### PR TITLE
update to pep8

### PR DESCRIPTION
**Pull request by zhaochenli**

### CODE REVIEW

File: xpdacq/beamtimeSetup.py [hunk 1]
```diff
@@ -33,8 +33,8 @@
 DATA_DIR = rs_fn('xpdacq', 'data/')
 
 
-
-def _start_beamtime(PI_last, saf_num, experimenters=[],
+def _start_beamtime(
```
Comment (CJ-Wright) [inline xpdacq/beamtimeSetup.py]: Why was this broken up over two lines?

File: xpdacq/beamtimeSetup.py [hunk 2]
```diff
@@ -63,7 +64,7 @@ def _start_beamtime(PI_last, saf_num, experimenters=[],
         src = os.path.join(DATA_DIR, 'Ni24.D')
         dst = os.path.join(glbl_dict['usrAnalysis_dir'], 'Ni24.D')
         shutil.copy(src, dst)
-        beamline_config = _load_beamline_config(glbl['blconfig_path'], test=test)
```
Comment (CJ-Wright) [inline xpdacq/beamtimeSetup.py]: Why was this line removed?

File: xpdacq/999-load.py [hunk 3]
```diff
@@ -17,36 +17,36 @@
...
 bt = start_xpdacq()
 if bt is not None:
     print("INFO: Reload beamtime objects:\n{}\n".format(bt))
 if reload_glbl_dict is not None:
     _set_glbl(glbl, reload_glbl_dict)
 
-# import necessary modules
-from xpdacq.xpdacq import *
-from xpdacq.beamtime import *
-from xpdacq.utils import import_sample_info
```
Comment (chiahaoliu) [inline xpdacq/999-load.py]: I wouldn't make any change to the import sequence in ``999-load.py`` unless it's carefully tested in simulation mode and at the beamline. This sequence is motivated by name-space management and serialization of ``glbl`` and other beamtime objects.

File: xpdacq/beamtime.py [hunk 4]
```diff
@@ -456,13 +459,18 @@ def statTramp(dets, exposure, Tstart, Tstop, Tstep, sample_mapping, *,
...
+
+# stream_name='primary'
+
 
 register_plan('ct', ct)
 register_plan('Tramp', Tramp)
 register_plan('tseries', tseries)
 register_plan('Tlist', Tlist)
-#register_plan('statTramp', statTramp)
+
```
Comment (CJ-Wright) [inline xpdacq/beamtime.py]: Does PEP8 require these extra lines?

File: xpdacq/beamtimeSetup.py [hunk 5]
```diff
@@ -53,7 +52,8 @@ def _start_beamtime(PI_last, saf_num, experimenters=[],
     elif len(f_list) == 0:
         _make_clean_env()
         print("INFO: initiated requried directories for experiment")
-        bt = Beamtime(PI_last, saf_num, experimenters,
+        bt = Beamtime(
```
Comment (CJ-Wright) [inline xpdacq/beamtimeSetup.py]: Please put some of the args on this line.

File: xpdacq/beamtimeSetup.py [hunk 6]
```diff
@@ -128,7 +128,8 @@ def load_beamtime(directory=None):
         directory = glbl_dict['yaml_dir']  # leave room for multi-beamtime
     known_uids = {}
     beamtime_fn = os.path.join(directory, 'bt_bt.yml')
-    sample_fns =  [fn for fn in
+    sample_fns = [
```
Comment (CJ-Wright) [inline xpdacq/beamtimeSetup.py]: Does PEP8 require this?

**Comment by sbillinge** [comment]
shall we close this, or merge it?

**Comment by CJ-Wright** [comment]
At this point I think we should just run `black -l 79 .` on everything and be done with it.

**Comment by sbillinge** [comment]
that's what I thought....

**Comment by CJ-Wright** [comment]
[Black](https://github.com/ambv/black) isn't perfect but it is mostly self consistent.
